src/pfe/portal/modules/User.js

```diff
@@ -737,8 +737,10 @@ module.exports = class User {
 
     // Only update the Kube secret when running in K8s
     let isServiceAccountPatched = false;
+    let serviceAccountPatchData;
     if (global.codewind.RUNNING_IN_K8S) {
-      isServiceAccountPatched = await this.updateServiceAccountWithDockerRegisrySecret();
+      serviceAccountPatchData = await this.updateServiceAccountWithDockerRegisrySecret();
```

Comment:
Can you please add some comments to the description of the `setupRegistrySecret` function that explains the usage of this function for the non-k8s cases? I'm sure others would wonder in the future.

Reply:
done